### DISCUSSION TITLE
Retain existing dossier notes

### DISCUSSION THREAD

**Discussion by trs4ece** [BlackAsLight/DocScripts#4] [Ideas]
I like the way that you persist settings through the dossier. However, some people keep other notes in the dossier. Could you retain any rows not related to DocScripts?

Example Dossier:

> KOS:
> https://politicsandwar.com/nation/id=146731
> https://politicsandwar.com/nation/id=12118
> 
> Repay $300M before 7/11
> [docscripts]185034019,"a924f76c8b235d57f692c4a279e78cf9e64",{"Doc_MinResource_Food":2500}[/docscripts]

**Comment by BlackAsLight** [comment]
It should definitely be possible. I'll look into it when I get to rewriting it in tsx.
